Felienne/hedy
# Localize quizes #777

**Description**

- Add localization for quizzes
- Consolidated quiz data files to language files
- Update quizzes json schema

**Fix for**

#777 

**How to test**

- Complete the quiz for level 1 in English and Dutch 
- Try reaching the max number of attempts in English and Dutch
- See whether it is ok to show the End tab (which leads to quizzes) for languages that have no adventure file, e.g. Italian
- While the files are yaml consolidated, the content is not changed, which means that there might be a mismatch between the levels provided in the end adventure and the quiz yaml data 

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
